### ISSUE TITLE
Multi data path

### ISSUE THREAD

**Issue by moreOver0** [ClickHouse/ClickHouse#1460]
There is `<path></path>` in config.xml to specify the data directory, so I'm wondering can we put tables/databases in different directory? For example, database A in `/mnt/ssd`, database B in `/mnt/hdd`.

**Comment by valyala** [comment]
Just use [symbolic links](https://en.wikipedia.org/wiki/Symbolic_link).

Suppose you have `<path>/opt/clickhouse/</path>`. Then create symbolic links in `/opt/clickhouse/data/` for every db you need:
```
ln -s /mnt/ssd/fastdb /opt/clickhouse/data/fastdb
ln -s /mnt/hdd/slowdb /opt/clickhouse/data/slowdb
```
The actual data for databases will be stored in `/mnt/ssd/fastdb` and `/mnt/hdd/slowdb`

**Comment by sinlang** [comment]
@valyala is there any document to introduce how clickhourse using multi disk? how to assign and balance between disk;

> Just use [symbolic links](https://en.wikipedia.org/wiki/Symbolic_link).
> 
> Suppose you have `<path>/opt/clickhouse/</path>`. Then create symbolic links in `/opt/clickhouse/data/` for every db you need:
> 
> ```
> ln -s /mnt/ssd/fastdb /opt/clickhouse/data/fastdb
> ln -s /mnt/hdd/slowdb /opt/clickhouse/data/slowdb
> ```
> The actual data for databases will be stored in `/mnt/ssd/fastdb` and `/mnt/hdd/slowdb`

**Comment by Tasselmi** [comment]
> Just use [symbolic links](https://en.wikipedia.org/wiki/Symbolic_link).
> 
> Suppose you have `<path>/opt/clickhouse/</path>`. Then create symbolic links in `/opt/clickhouse/data/` for every db you need:
> 
> ```
> ln -s /mnt/ssd/fastdb /opt/clickhouse/data/fastdb
> ln -s /mnt/hdd/slowdb /opt/clickhouse/data/slowdb
> ```
> The actual data for databases will be stored in `/mnt/ssd/fastdb` and `/mnt/hdd/slowdb`

it seems  this does not work .
I created symbolic link in clickhouse/data/test 
but when I create tables under db-test , it created table files in clickhouse/data/test  ~~

**Comment by filimonov** [comment]
Implemented in 19.15
